src/handlers/web3.js

```diff
@@ -329,12 +363,13 @@ export const getTransferNftTransaction = async transaction => {
   const { from } = transaction;
   const contractAddress = get(transaction, 'asset.asset_contract.address');
   const data = getDataForNftTransfer(from, recipient, transaction.asset);
+  const gasParams = getTransactionGasParams(transaction);
```

Comment:
we can remove the comment (lines 356-360) as it is outdated and will be TS'd shortly